app/helpers/application_helper/toolbar/ems_containers_center.rb

```diff
@@ -29,7 +29,7 @@ class ApplicationHelper::Toolbar::EmsContainersCenter < ApplicationHelper::Toolb
           t = N_('Create Containers Provider'),
           t,
           :data   => {'function' => 'miqCallAngular', 'function-data' => '{ "name": "showListener", "args": [] }'},
-          :hidden => ContainerDeploymentService.hide_deployment_wizard?),
+          :hidden => hide_deployment_wizard?),
```

Comment:
This will cause the function to be called once at the first load of this toolbar.

The correct pattern would be to have this evaluated at the display of the toolbar as we do with other (most?) buttons.

But his is not introduces by this PR. It's just more exposed because of the private method.

Hmm, does this work? I mean the toolbar definition is done in class context in class load time and the private method is defined on an instance. Am I missing something?